rs/sns/init/src/lib.rs

```diff
@@ -563,25 +563,6 @@ impl SnsInitPayload {
         Ok(governance)
     }
 
-    #[cfg(feature = "test")]
-    fn maybe_test_balances(&self) -> Vec<(Account, u64)> {
```

Comment:
I wonder if we could move this logic that creates these extra accounts to NNS Governance's "test" canister...?  

Or do we do the deployments directly when this test feature flag is enabled (that is, through sns cli directly)?

Reply:
AFAIK we don't use this at all, although the comment claims we use it in SNS Testing. Either way, I don't think we need this, anyone who needs tokens from a test SNS can use mint_tokens

Reply:
I haven't tested SNSDEMO or nns-dapp, but I spoke with david and he says it will be fine.